app/javascript/packs/application.js

```diff
@@ -16,3 +16,7 @@
 // const imagePath = (name) => images(name, true)
```

Comment:
- [ ] please don't push any commented-out code they will make your codebase look unorganised and harder to read.